checkpoint/cp-resource-management/README.md

```diff
@@ -80,6 +80,7 @@ The following tables list the configurable parameters of this chart and their de
 | `credentials.secret`                                       | CloudGuard API Secret                                           | `CHANGEME`                                       |
 | `credentials.user`                                         | CloudGuard API ID                                               | `CHANGEME`                                       |
 | `clusterID`                                                | Cluster Unique identifier in CloudGuard system                  | `CHANGEME`                                       |
+| `datacenter`                                               | CloudGuard datacenter (usea1 (default), euwe1 etc.)             | `usea1`                                          |
```

Comment:
Is there a public resource where all the regions are listed, in the form that we expect here?

Reply:
I don't think so. We are being referred to internal wiki. I would expect customers to use values they get in UI. Then moving to automation after manually onboarding some clusters.
Will try to check again